l10n_it_fatturapa/readme/CONFIGURE.rst

```diff
@@ -5,6 +5,8 @@
 * Gli altri devono avere il codice destinatario o la PEC
 * Configurare le imposte riguardo a "Natura non imponibile", "Riferimento normativo" ed "Esigibilità IVA"
 * Configurare i dati della fattura elettronica nella configurazione della contabilità, dove necessario
+* Opzionalmente configurare lo style della preview della fattura elettronica, selezionando "Preview Format Style" in "Configurazione Contabilità"
```

Comment:
`style` > `stile`
`Preview Format Style` > `Stile del formato di anteprima` (poi da tradurre su weblate)

PS: @robyf70 quando carichi delle modifiche che sono pronto per essere revisionate dovresti anche mandare un messaggio, altrimenti non ne veniamo informati.
Grazie

Reply:
@eLBati Ok!

Reply:
@eLBati è tutto ok su questa PR? C'è qualcosa da rivedere?

Reply:
Proponevo questo: https://github.com/OCA/l10n-italy/pull/1036#discussion_r269130953

Reply:
@eLBati Perdonami Lorenzo, non capisco cosa vorresti inserire

Reply:
Proponevo di modificare `style della preview` in `stile dell'anteprima` e `Preview Format Style` in `Stile del formato di anteprima`.

Per usare termini italiani nella parte di documentazione in italiano.

Reply:
@eLBati Ok! Fatto! Ho appena pushato